docker-compose/monitoring/src/config.ts

```diff
@@ -0,0 +1,32 @@
+import { type BitcoinAddress } from '@rsksmart/rsk-monitor'
+
+
+/**
+ * Uncomment this to monitor testnet addresses
+ */
+export const MONITORED_ADDRESSES: BitcoinAddress[] = [
+  { address: 'mwEceC31MwWmF6hc5SSQ8FmbgdsSoBSnbm', alias: 'testnet1' },
+  { address: 'mvL2bVzGUeC9oqVyQWJ4PxQspFzKgjzAqe', alias: 'testnet2' },
+  { address: 'tb1q7hec37mcmfk6hmqn67echzdf8zwg5n5pqnfzma', alias: 'testnet3' }
+]
+
+export const MONITOR_CONFIG = {
+  pollingIntervalSeconds: 10,
+  monitorName: 'bitcoin-balance-monitor',
+  network: 'testnet' as 'mainnet' | 'testnet'
+} 
+
+
+/**
```

Comment:
Remove this comment maybe?

Reply:
I wanted to leave it there so it is easier to configure. This comment is even in the documentation as an option/example of how to configure mainnet addresses

Reply:
Changed.